lib/token.js

```diff
@@ -212,10 +221,11 @@ function getDefaultOAuthParams(sdk, oauthOptions) {
   oauthOptions = util.clone(oauthOptions) || {};
 
   var defaults = {
+    grantType: 'implicit',
     clientId: sdk.options.clientId,
     redirectUri: sdk.options.redirectUri || window.location.href,
     responseType: 'id_token',
-    responseMode: 'okta_post_message',
+    responseMode: 'fragment',
```

Comment:
Why are we moving away from `okta_post_message`? Does our new PKCE flow not support this?

Reply:
This is changing the *default* responseMode. calling getWithPopup() will set this to 'okta_post_message'. I have removed some complex logic involving setting a responseMode dynamically: https://github.com/okta/okta-auth-js/pull/205/files/bbcf753b76c571f1e342441e9b05bcd69468564d#diff-398ed3b014436a5204583323ea29320bL475

Reply:
Please ensure this is outlined as a **breaking change** in the release notes.

Reply:
I have removed this breaking change, and am continuing to use 'okta_post_message' as default